gateway/ln-gateway/src/cln.rs

```diff
@@ -44,7 +44,8 @@ pub struct Htlc {
 #[derive(Clone, Serialize, Deserialize, Debug)]
 pub struct Onion {
     pub payload: String,
-    pub short_channel_id: String,
+    #[serde(default)]
+    pub short_channel_id: Option<String>,
```

Comment:
@okjodom Be sure to do the same for the new GW.

Reply:
Removed all other fields because we don't use them?

Reply:
For a gateway we're running right,`short_channel_id` is occasionally missing. This leads to a panic because `serde_json` expects it to be there.

Removing other fields because we don't use them and don't want to panic in a similar way in case they're not there.